app/controllers/spotlight/catalog_controller.rb

```diff
@@ -109,6 +109,16 @@ def make_public
       end
     end
 
+    def manifest
+      _, @document = fetch params[:id]
```

Comment:
This is a super minor nitpick, but since we're simply rending json inline below the `@document` ivar is pointless.  A local variable would suffice since we're not rending views.

If you're in there and feel like changing it, by all means, but I wouldn't hold this PR up for it.